Task2-5/Task2-5/Program.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+
+namespace Task2_5
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.Write("Enter the lengths of the sides of the triangle.\n\nSide 1: ");
+            bool firstSuccessInput = float.TryParse(Console.ReadLine(), out float firstSide);
+
+            Console.Write("Side 2: ");
+            bool secondSuccessInput = float.TryParse(Console.ReadLine(), out float secondSide);
+
+            Console.Write("Side 3: ");
+            bool thirdSuccessInput = float.TryParse(Console.ReadLine(), out float thirdSide);
+
+            if (firstSuccessInput && secondSuccessInput && thirdSuccessInput)
+            {
+                if (firstSide + secondSide > thirdSide && firstSide + thirdSide > secondSide && secondSide +
+                    thirdSide > firstSide)
```

Comment:
Don't use it here.

Reply:
Do I need to solve this task in a different way or do I have problems with line break?